ui/raidboss/data/05-shb/dungeon/matoyas_relict.txt

```diff
@@ -0,0 +1,59 @@
+### Matoya's Relict
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0 "--Reset--" sync /00:0839:.*is no longer sealed/ window 100000 jump 0
+
+### Mudman
+# -p 547F:1011
+# -ii 368 5481 5487 548E 548F 5490 5492
+1000.0 "--sync--" sync /00:0839:Clayclot Cauldron will be sealed off/ window 1000,0
+# Loop
+1011.0 "Hard Rock" sync /:Mudman:547F:/ duration 6 window 0,70
+1020.2 "Petrified Peat" sync /:Mudman:5480:/
+1032.3 "Peat Pelt" sync /:Mudman:5482:/
+1041.4 "Rocky Roll" sync /:Mud Bubble:548[34]:/
+1050.7 "Brittle Breccia" sync /:Mudman:548D:/
+1051.1 "Rocky Roll" sync /:Mud Bubble:548[35]:/
+1066.7 "Stone Age" sync /:Mudman:5491:/ duration 5
+
+
+### Nixie
+# -p 598F:2009.2
+# -ii 598E 5990 5992 5993 5994 5995 5996 5BB9
+2000.0 "--sync--" sync /00:0839:Clearnote Cauldron will be sealed off/ window 2000,0
+
+2009.2 "Crash-Smash" sync /:Nixie:598F:/ duration 6
+2023.4 "Shower Power" sync /:Nixie:5991:/
+2044.1 "Pitter-Patter" sync /:Nixie:5988:/
+2072.2 "Sea Shanty" sync /:Nixie:598[AC]:/
+2092.4 "Shower Power" sync /:Nixie:5991:/
+2110.5 "Crash-Smash" sync /:Nixie:598F:/ duration 6
+2130.7 "Shower Power" sync /:Nixie:5991:/
+2134.9 "Splish-Splash" sync /:Nixie:598D:/
+2160.0 "Shower Power" sync /:Nixie:5991:/
+
+### Mother Pirxie
+# -p 5913:3010.0
+# -ii 368 4A8F 5912 5914 5917 5918 591E 591F 5923 5925
+3000.0 "--sync--" sync /00:0839:Woebegone Workshop will be sealed off/ window 3000,0
+
+3010.0 "Tender Loin" sync /:Mother Porxie:5913:/
+3028.2 "Huff And Puff" sync /:Mother Porxie:5919:/
+3030.3 "Medium Rear" sync /:Mother Porxie:591D:/
+3044.4 "Meat Mallet" sync /:Mother Porxie:5916:/
+3049.3 "Barbeque" sync /:Mother Porxie:5B23:/
+3071.5 "To A Crisp" sync /:Mother Porxie:5924:/
+3085.6 "Minced Meat" sync /:Mother Porxie:5911:/ duration 6
+3101.9 "Buffet" sync /:Aeolian Cave Sprite:5926:/
+3107.2 "Buffet" sync /:Aeolian Cave Sprite:5926:/
+3111.8 "Explosion" sync /:Aeolian Cave Sprite:4E34:/
+3122.9 "Huff And Puff" sync /:Mother Porxie:591B:/
+3125.0 "Medium Rear" sync /:Mother Porxie:591D:/
+3141.2 "Open Flame" sync /:Mother Porxie:5922:/
+3153.3 "Minced Meat" sync /:Mother Porxie:5911:/ duration 6
+3165.4 "Tender Loin" sync /:Mother Porxie:5913:/
+3179.5 "Tender Loin" sync /:Mother Porxie:5913:/
+3199.0 "Huff And Puff" sync /:Mother Porxie:5919:/
+3201.1 "Medium Rear" sync /:Mother Porxie:591D:/
```

Comment:
If you don't know the loop here, can you leave a `# ???` or a `# TODO: find the loop` so that it's clear that this is incomplete?

Reply:
```
# -ii 4A8F 5912 5914 5917 5918 591E 591F 5923 5925

0 "Start" sync /00:0839:The Woebegone Workshop will be sealed off/ window 0,1
11.3 "Tender Loin" sync /:Mother Porxie:5913:/
29.5 "Huff And Puff" sync /:Mother Porxie:5919:/
31.6 "Medium Rear" sync /:Mother Porxie:591D:/
45.7 "Meat Mallet" sync /:Mother Porxie:5916:/
45.7 "--untargetable--"
50.4 "Barbeque" sync /:Mother Porxie:5B23:/
72.4 "To A Crisp" sync /:Mother Porxie:5924:/
74.4 "--targetable--"
86.5 "Minced Meat" sync /:Mother Porxie:5911:/ window 50,10
93.6 "--untargetable--"

96.6 "Huff And Puff" sync /:591A:Mother Porxie/ window 96.6,5 duration 49.70
101.6 "Buffet" sync /:Aeolian Cave Sprite:5926:/
107.3 "Buffet" sync /:Aeolian Cave Sprite:5926:/
112.8 "Buffet" sync /:Aeolian Cave Sprite:5926:/
118.4 "Buffet" sync /:Aeolian Cave Sprite:5926:/
124.0 "Buffet" sync /:Aeolian Cave Sprite:5926:/
129.6 "Buffet" sync /:Aeolian Cave Sprite:5926:/
130.3 "Huff And Puff Enrage?" sync /:Mother Porxie:591B:/ window 50,5

132.3 "--targetable--"
132.4 "Medium Rear" sync /:Mother Porxie:591D:/
148.5 "Open Flame" sync /:Mother Porxie:5922:/
160.6 "Minced Meat" sync /:Mother Porxie:5911:/
172.7 "Tender Loin" sync /:Mother Porxie:5913:/
186.8 "Tender Loin" sync /:Mother Porxie:5913:/
207.1 "Huff And Puff" sync /:Mother Porxie:5919:/
209.2 "Medium Rear" sync /:Mother Porxie:591D:/
225.3 "Open Flame" sync /:Mother Porxie:5922:/
237.5 "Minced Meat" sync /:Mother Porxie:5911:/ jump 160.6
249.6 "Tender Loin" sync /:Mother Porxie:5913:/
263.7 "Tender Loin" sync /:Mother Porxie:5913:/
281.9 "Huff And Puff" sync /:Mother Porxie:5919:/
```

Sorry for the text dump, but this is the entirety of the Porxie encounter. The intermission needs some work, but should be basically correct. The Huff And Puff *seems* to be a consistent cast time. Buffet should probably not be synced since the Sprite should die early on.

Reply:
Thanks for all of this!

> Buffet should probably not be synced since the Sprite should die early on.

Yeah, I think so.
But perhaps we can add a sync entry like
`"--sync--" sync /:Aeolian Cave Sprite:4E34:/`
before targetable lines, so can we know the targetable timing exactly.

Reply:
Oh, the timing there is exact, I inserted it with the `-it` flag. The abilities before each one should be good enough for syncs.

Reply:
Ok, I'll merge yours directly. Thanks!
Having a bussiness trip today so I can only work on a ipad, impossible to test it in dungeon.😂

Reply:
Oof, what a time to have to do that. Yes, this is functional and should be safe to merge.